standards/core/store.md

```diff
@@ -1,208 +1,331 @@
 ---
-title: 13/WAKU2-STORE
-name: Waku v2 Store
+title: WAKU2-STORE
```

Comment:
In my opinion, the name being called Store without v3 is good. The version number could be added within the spec, e.g. within the protocol identifier or a statement that links the previous version could also mention the number of the current version.

Reply:
Indeed. This is just the new version of the Store protocol. The "v3" was just a convenient way to reference this version, but shouldn't be considered as part of the protocol signifier. It's only really reflected in the libp2p protocol ID.